sql/ddl.mdx

```diff
@@ -52,14 +52,41 @@ a Kafka connection called my_kafka and a Kafka topic order_topic:
 ```sql
 CREATE TABLE orders (
   customer_id INT,
-  order_id INT)
- WITH (
+  order_id INT,
+  date_string TEXT,
+  event_time TIMESTAMP GENERATED ALWAYS AS (CAST(date_string as TIMESTAMP)),
+  watermark TIMESTAMP GENERATED ALWAYS AS (event_time - INTERVAL '15' SECOND)
+) WITH (
   connection = 'my_kafka',
   topic = 'order_topic',
-  format = 'json'
+  format = 'json_schema_registry',
```

Comment:
this should be `serialization_mode` -- also does `json_schema_registry` work here? I think to take advantage of the schema registry you have to create it in the UI.

Reply:
It should work, in that it will just trim the first 5 bytes and then try to deserialize.

Reply:
yeah, I guess it's more that it's weird that you'd use the SQL interface which doesn't really support the schema registry for kafka topics that have schema data